### DISCUSSION TITLE
Welcome to schemaorg Discussions!

### DISCUSSION THREAD

**Discussion by danbri** [schemaorg/schemaorg#2784] [General]
## Welcome!

  We’re using Discussions as a place to connect with other members of our community. Please consider having an discussion here rather than opening a formal issue. Issues are requests for a small team to work on something, and we have 100s of them already! Sometimes it's better to chat first...:)

**Comment by mignon1915** [comment]
Hi Dan & everybody else!

My question is around extensions to schema.org and how you've solved for conflicting definitions for properties. Is it simply a combination of the fact that each property is uniquely identified and some sort of approval process that gets triggered based on the 'pending' label? How do discussions around this happen when a new extension is being negotiated? What is the thinking around this in regards to external extensions? Thanks very much.

**Reply by thadguidry** [reply]
Hi @mignon1915 Create a New Discussion so we can talk about that and respond.
